payments-core/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -245,14 +243,12 @@ internal class PaymentSheetViewModel @Inject internal constructor(
                         PaymentMethod.Type.fromCode(it)
```

Comment:
optional: the logic here is identical to `DefaultFLowControllerInitializer#createWithCustomer`, where a `StripeIntent` is in and a `List<PaymentMethod>` is out, I wonder could we combine them somewhere?

Reply:
The logic is very similar but there are some slight differences that make it harder to combine them further than this, like `DefaultFLowControllerInitializer` needs the list of payment method types to use in the configuration.